Projects/Modules/Documentarian.MicrosoftDocs/CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ release_links:
 
 ## Unreleased
 
+- Added Get-CmdletXref
```

Comment:
I think we can help folks understand a bit about the change and link them right to the new reference docs:

```suggestion
- Added [`Get-CmdletXref`] to make it simpler to add links to cmdlet reference docs.

[`Get-CmdletXref`]: /modules/microsoftdocs/reference/cmdlets/get-cmdletxref/
```